src/clj/game/cards/events.clj

```diff
@@ -522,6 +522,32 @@
     :msg "look at the top 4 cards of their Stack and add 1 of them to their Grip"
     :effect (effect (move target :hand) (shuffle! :deck))}
 
+   "Falsified Credentials"
+   {:prompt "Choose a type"
+    :choices ["Agenda" "Asset" "Upgrade"]
+    :effect (effect
+             (resolve-ability
```

Comment:
`resolve-ability` creates a new `eid` for the effect it resolves, and that `eid` will be marked completed when the chained ability completes, which is great... but we also need the `eid` of the original parent ability to complete, which means it needs to be bound to a name and then passed to `effect-completed` by hand. That would be a huge pain in the ass, so instead we have `continue-ability`, which is a drop-in replacement for `resolve-ability`, except it gives the current eid's ability to the new ability, so that the completion of the new ability also marks the completion of the original ability.